cmd/admin-router.go

```diff
@@ -29,12 +29,20 @@ func registerAdminRouter(mux *router.Router) {
 	// Admin router
 	adminRouter := mux.NewRoute().PathPrefix("/").Subrouter()
 
-	/// Admin operations
+	/// Service operations
 
 	// Service status
 	adminRouter.Methods("GET").Queries("service", "").Headers(minioAdminOpHeader, "status").HandlerFunc(adminAPI.ServiceStatusHandler)
 	// Service stop
 	adminRouter.Methods("POST").Queries("service", "").Headers(minioAdminOpHeader, "stop").HandlerFunc(adminAPI.ServiceStopHandler)
 	// Service restart
 	adminRouter.Methods("POST").Queries("service", "").Headers(minioAdminOpHeader, "restart").HandlerFunc(adminAPI.ServiceRestartHandler)
+
+	/// Lock operations
+
+	// List Locks
+	adminRouter.Methods("GET").Queries("lock", "", "bucket", "{bucket:.+}", "prefix", "{prefix:.+}", "older-than", "{older-than:.*}").Headers(minioAdminOpHeader, "list").HandlerFunc(adminAPI.ListLocksHandler)
```

Comment:
I think it is better to avoid filtering requests here by bucket, prefix and older-than, so we can return a proper error message when lock is specified but other variables are not.. otherwise the request will be routed somewhere else, usually to object API